test/e2e/components/vl-pill.js

```diff
@@ -1,10 +1,48 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
 
-class VlPill extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+class VlPill extends VlElement {
+    async isClosable() {
+        return this.hasAttribute('closable');
     }
 
+    async isCheckable() {
+        return this.hasAttribute('checkable');
+    }
+
+    async isChecked() {
+        return await this.driver.executeScript('return arguments[0].checked', this);
+    }
```

Comment:
is dit nodig? eikes dan.
evt dan in VlElement te steken?

Reply:
Ja, getAttribute geeft wel een property terug van het element, maar omdat checked een getter is, is dat geen gewone property. De executeScript is daarom nog nodig zodat de getter geevalueerd wordt.
Anderen hun mening? In VlElement steken of zo laten?

Reply:
Ik zou het niet in de `VlElement` steken omdat bij een input dit wel gaat werken. Het is een afwijkende implementatie die hier dan goed staat.